src/main/java/com/mercadolibre/bootcampmelifrescos/model/AgentModel.java

```diff
@@ -0,0 +1,21 @@
+package com.mercadolibre.bootcampmelifrescos.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "agents")
+public class AgentModel {
```

Comment:
Vocês acham que vale a pena adicionar esse model em todas as classes? To acostumado a deixar só o nome da entidade mesmo 😄 

```suggestion
public class Agent {
```

Reply:
Não, pode tirar o model de todas.

Reply:
Eu coloco basicamente pra poder ficar mais fácil da pessoa entender se é um model, ou se é um controller, tipo pra ficar mais legível

Reply:
resolvido